source/agora/node/FullNode.d

```diff
@@ -521,9 +521,18 @@ public class FullNode : API
     ***************************************************************************/
 
     protected void onAcceptedBlock (const ref Block block,
-        bool validators_changed) @safe
+        bool validators_changed) @safe nothrow
     {
-        this.pushBlock(block);
+        try
+        {
+            this.pushBlock(block);
+        }
+        catch (Exception ex)
+        {
+            log.error("Error pushing block: {}" ~
+                "Block: {}", ex.msg, block);
```

Comment:
```suggestion
                "Block: {}", ex.msg, prettify(block));
```
I don't know if the information needed for debugging should look pretty.

Reply:
Yes, that is a good idea. It is also used in other `log`s.